### DISCUSSION TITLE
Publishing failed :| Something went wrong

### DISCUSSION THREAD

**Discussion by wkolov** [codemagic-ci-cd/codemagic-docs#1942] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Start build
2. Wait
3. Get the build failed with the message "Publishing failed :| Something went wrong" for the Play Store step. The App Store step works good.

### Expected results

The built AAB artifact is uploaded to the Play Store as a draft release OR I at least can figure out, what the problem is. The publishing has worked earlier, but the we've got problems accessing GitHub. After we fixed the GitHub access, publishing doesn't work any more.

### Actual results

The AAB file is built and can be uploaded to the Play Store manually, but the automatic publishing doesn't work.

### Build id (optional)

64cbc0ab8a465846dacd5ebc

**Comment by dtrdic** [comment, marked as answer]
hey @wkolov could you please check if the service account or the JSON key has expired or been deleted?
I can see this error HttpAccessTokenRefreshError: invalid_grant: Invalid grant: account not found. 
Try exporting a new .json file and add it as an environment variable.
Make sure you're following this [guide](https://docs.codemagic.io/flutter-publishing/publishing-to-google-play/)

**Reply by wkolov** [reply]
Thank you for your reply. I assumed something like this. Wouldn't it be better to display the message directly instead of "Something went wrong"?
I have to ask the admin of our Google Play account to export the new credentials file to verify the solution. It can last.

**Reply by dtrdic** [reply]
Agree, i forwarded your suggestion. No problem, if you encounter any further issues or have any questions along the way, don't hesitate to reach out

**Comment by d7p** [comment]
I had the same issue. Generating a new key and updating the environment variable solved the issues. 

+1 for the error message needs to be better.

**Comment by wkolov** [comment]
It works again with a new JSON file
